packages/protoplugin/src/ecmascript/gencommon.ts

```diff
@@ -58,7 +58,7 @@ export function makeFilePreamble(
   };
   writeLeadingComments(file.getSyntaxComments());
   builder.push(`// @generated by ${pluginName} ${pluginVersion}`);
-  if (parameter !== undefined) {
+  if (parameter !== undefined && parameter !== "") {
```

Comment:
We never pass `undefined` to this function. Please change line 37 to be `parameter: string`, without `| undefined`.

Then this can be:

```suggestion
  if (parameter !== "") {
```